.github/workflows/push-module-test.yaml

```diff
@@ -46,3 +42,15 @@ jobs:
           bundleId: test-app1
           version: 0.1.1-SNAPSHOT
           metadataBucket: gs://bw-ee-labs-slipstream-github-action-test
+
+  build:
```

Comment:
There's generally a lot of duplication on these build steps. Might make more sense to refactor them out as a composite step, and then reference them with a `uses: ...` so they can all be maintained at the same time

Reply:
This was exactly my first thought as well, I would probably run test and builds in a matrix. But I would like keep it separate PR, which is mainly done to resolve deprecations. If that's ok with you I would like to keep as it is for this PR.

Reply:
Yeah sure that sounds fine :)